src/Language/PureScript/Errors.hs

```diff
@@ -713,7 +713,7 @@ prettyPrintSingleError (PPEOptions codeColor full level showDocs relPath) e = fl
     renderSimpleErrorMessage (DeprecatedFFICommonJSModule mn path) =
       paras [ line $ "A CommonJS foreign module implementation was provided for module " <> markCode (runModuleName mn) <> ": "
             , indent . lineS $ path
-            , line "CommonJS foreign modules are deprecated and won't be supported in the future."
+            , line "CommonJS foreign modules are no longer supported."
```

Comment:
```suggestion
            , line "CommonJS foreign modules (imports using `require`) are no longer supported, please use `import` instead."
```

Reply:
Or something like this could be beneficial to people not familiar with the term CommonJS and seems more immediately actionable to me.

Reply:
I think this is a valid and useful change. However changing the error message always requires updating the tests. So before I do this, I would like to ask here if you all agree on this or have more proposals on how to improve this error message. Thx :-)

Reply:
> Or something like this could be beneficial to people not familiar with the term CommonJS and seems more immediately actionable to me.

Isn't that what the [documentation page for DeprecatedFFICommonJsModule](https://github.com/purescript/documentation/blob/master/errors/DeprecatedFFICommonJSModule.md) is for?

I don't think this change needs to be done.

Reply:
I think it's worth mentioning what should be used instead, but `imports using require`  isn't the whole story (what about exports using the exports/module object?). I think we should say:

> CommonJS foreign modules are no longer supported. Use native JavaScript/ECMAScript module syntax instead.

Reply:
Ok I updated it to @natefaubion 's proposal.